src/graphnet/models/task/reconstruction.py

```diff
@@ -12,6 +12,8 @@ class AzimuthReconstructionWithKappa(Task):
     """Reconstructs azimuthal angle and associated kappa (1/var)."""
 
     # Requires two features: untransformed points in (x,y)-space.
+    default_target_labels = ["azimuth"]
+    default_output_labels = ["azimuth_pred", "azimuth_kappa"]
```

Comment:
To be consistent with suggestion in `standard_model.py`. Similar below.
```suggestion
    default_prediction_labels = ["azimuth_pred", "azimuth_kappa"]
```